app/src/pages/inside/testCaseLibraryPage/historyOfActions/historyOfActions.scss

```
@@ -14,11 +14,110 @@
+
+    /* ui-kit: row has min-width:fit-content; cells use overflow:hidden + nowrap + ellipsis */
+    :global([class*='table-row-content']) {
+      min-width: 0 !important;
+    }
+
+    :global([class*='table-row-content']) > :global([class*='_table-cell_']) {
+      min-width: 0 !important;
+      overflow: visible !important;
+      white-space: normal !important;
+      text-overflow: clip !important;
+      word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)